File: uri/C/beginner/1048.c
#include <stdio.h>

int main()
{

	double salario, nsalario, Reajuste;

	scanf("%lf", &salario);

	if(0 < salario && salario <= 400){
		nsalario = salario * 1.15;
		Reajuste = salario * 0.15;
		printf("Novo salario: %.2lf\n", nsalario);
		printf("Reajuste ganho: %.2lf\n", Reajuste);
		printf("Em percentual: 15 %%\n");
	}

	if(400 < salario && salario <= 800){
		nsalario = salario * 1.12;
		Reajuste = salario * 0.12;
		printf("Novo salario: %.2lf\n", nsalario);
		printf("Reajuste ganho: %.2lf\n", Reajuste);
		printf("Em percentual: 12 %%\n");
	}

	if(800 < salario && salario <= 1200){
		nsalario = salario * 1.10;
		Reajuste = salario * 0.10;
		printf("Novo salario: %.2lf\n", nsalario);
		printf("Reajuste ganho: %.2lf\n", Reajuste);
		printf("Em percentual: 10 %%\n");
	}

	if(1200 < salario && salario <= 2000){
		nsalario = salario * 1.07;
		Reajuste = salario * 0.07;
		printf("Novo salario: %.2lf\n", nsalario);
		printf("Reajuste ganho: %.2lf\n", Reajuste);
		printf("Em percentual: 7 %%\n");
	}

	if(salario > 2000){
		nsalario = salario * 1.04;
		Reajuste = salario * 0.04;
		printf("Novo salario: %.2lf\n", nsalario);
		printf("Reajuste ganho: %.2lf\n", Reajuste);
		printf("Em percentual: 4 %%\n");
	}

	return 0;
}

File: uri/python/1048.py
# -*- coding: utf-8 -*-

salario = float(input())

novoSalario = 0
reajuste = 0

if salario >= 0 and salario <= 400.00:
	novoSalario = salario * 1.15
	reajuste = novoSalario - salario
	print('Novo salario: {:.2f}'.format(novoSalario))
	print('Reajuste ganho: {:.2f}'.format(reajuste))
	print('Em percentual: 15 %')

elif salario >= 400.01 and salario <= 800.00:
	novoSalario = salario * 1.12
	reajuste = novoSalario - salario
	print('Novo salario: {:.2f}'.format(novoSalario))
	print('Reajuste ganho: {:.2f}'.format(reajuste))
	print('Em percentual: 12 %')

elif salario >= 800.01 and salario <= 1200.00:
	novoSalario = salario * 1.10
	reajuste = novoSalario - salario
	print('Novo salario: {:.2f}'.format(novoSalario))
	print('Reajuste ganho: {:.2f}'.format(reajuste))
	print('Em percentual: 10 %')

elif salario >= 1200.01 and salario <= 2000.00:
	novoSalario = salario * 1.07
	reajuste = novoSalario - salario
	print('Novo salario: {:.2f}'.format(novoSalario))
	print('Reajuste ganho: {:.2f}'.format(reajuste))
	print('Em percentual: 7 %')

elif salario > 2000.00:
	novoSalario = salario * 1.04
	reajuste = novoSalario - salario
	print('Novo salario: {:.2f}'.format(novoSalario))
	print('Reajuste ganho: {:.2f}'.format(reajuste))
	print('Em percentual: 4 %')

